english/teams/team-ironman.md

```diff
@@ -0,0 +1,3 @@
+Team Ironman believes that superheroes need to be looked over by a governing body that decides when their services are needed. While this may suck for superheroes 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.